### ISSUE TITLE
[BUG] Stellar signing / transaction timebounds

### ISSUE THREAD

**Issue by MisterTicot** [trezor/trezor-firmware#582]
**Issue**

Trezor doesn't displays `timebounds.minTime` proprely, but the signature is
valid.

**Transaction Template**

```
{
  "source": "GAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWHF",
  "sequence": 0,
  "operations": [
    {
      "type": "setOptions"
    }
  ],
  "fee": 100,
  "timebounds": {
    "minTime": 1577836800,
    "maxTime": null
  }
}
```

**Device Display**

`Valid from [no restriction]`

**Comment by matejcik** [comment]
confirmed on T1. TT crashes on an operation that has a minimum bound but not a maximum.

**Comment by matejcik** [comment]
TT does not have a timebounds screen at all!

**Comment by MisterTicot** [comment]
Why did you change "Valid from" in "Valid to"? minTime means "Valid from", right?

**Comment by matejcik** [comment]
you'll also notice that the place I changed it refers to `timebounds_end`

**Comment by MisterTicot** [comment]
I see it now :)

**Comment by tsusanka** [comment]
Done for core in 61430d800e107d561f4be153e6ec32b025afcebe. It currently displays the timestamp which is not very user-friendly, but I have encountered some problems when trying to format the date. I have created a new issue for that #741. 

Also, it seems @matejcik has fixed this for legacy in https://github.com/trezor/trezor-firmware/commit/d48c5c645055840a685f46777fd9aa3520bdd8c8. So I am closing this.
